configuration/pih/scripts/global/zl.js

```diff
@@ -216,17 +181,18 @@ function setButtonsDisabled(val){
  * @param badDateInTheFutureMsg
  */
 function setUpObsWithObsDateTime(widgetId){
-    getField(widgetId + '.date').hide();
-    getField(widgetId + '.date').datepicker('option', 'maxDate', new Date());
-
-    getField(widgetId + '.value').change(function() {
-      const isChecked = getValue(widgetId + '.value');
-      if(isChecked) {
-        getField(widgetId + '.date').show();
-      }
-      else {
-        setValue(widgetId + '.date','')
-        getField(widgetId + '.date').hide();
-      }
-    })
+    if (getField(widgetId + '.date') && getField(widgetId + '.value')) {
```

Comment:
this is a follow-up to the last commit, to make this null-safe... we really shouldn't be initializing this unless we know the widgets exist, but are some cases where the syphilis question isn't shown on the form but the setup is still happening.